RN 37233 (#2070)
SkylineCommunications/dataminer-docs: user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/Visualizations/Available_visualizations/General/Stepper.md

     > [!NOTE]
     >

>      > - If the DOM history saving behavior of a module has been set to *Disabled* in [the `DomInstanceHistorySettings` object](xref:DOM_DomInstanceHistorySettings), the stepper component will not show a history of traversed states for any DOM instances. From DataMiner 10.3.11/10.4.0 onwards<!--RN 37233-->, instances without a history display the happy path from the initial state to the current state. In DataMiner 10.3.10, instances without a history calculate the path from the initial state to the current state as if they followed the standard flow, similar to a happy path.

Author: ```suggestion
     > - If the DOM history saving behavior of a module has been set to *Disabled* in [the `DomInstanceHistorySettings` object](xref:DOM_DomInstanceHistorySettings), the stepper component will not show a history of traversed states for any DOM instances. From DataMiner 10.3.11/10.4.0 onwards<!--RN 37233-->, instances without a history calculate the happy path from the initial state to the current state and then to the final state. In DataMiner 10.3.10, instances without a history calculate the happy path from the current state to the final state.
```

Is this more accurate then, @TVincke98?
